### DISCUSSION TITLE
Has any one tried this on MacOS apple silicon?

### DISCUSSION THREAD

**Discussion by limingchina** [fishaudio/fish-speech#351] [Q&A]
I saw the doc only mentions installation on Windows and Linux. Wonder if anyone has tried MacOS? I have an MacBook M3 pro and wonder if I can setup the lib and run the web UI.

**Comment by PortNumber53** [comment]
I just tried, and got
```
Expected elements.dtype() == test_elements.dtype() to be true, but got false. (Could this error message be improved? If so, please report an enhancement request to PyTorch.)
```
when clicking Generate. Unfortunately I cannot spend too much time figuring it out....   (M1 macbook pro Sequoia 15.1.1 (24B91) , in case it matters)

**Reply by p36348** [reply]
Same issue with M4 Max... It only appears on the web page, and there is no relevant information in the console.

**Comment by mr-callim** [comment]
I'm experiencing the same issue
